.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!-- Do NOT delete this template. Its great that you decided to contribute 
+to our project. :) Kindly fill the template accordingly! -->
+### Solves Issue # <number>
```

Comment:
change this to `fixes #<num>` or `closes #<num>`

Reply:
Done.